snap/info.go

```diff
@@ -65,3 +71,13 @@ type AppInfo struct {
 	Plugs   map[string]*PlugInfo
 	Slots   map[string]*SlotInfo
 }
+
+// StoreInfo provides specific information for a store hosted snap.
+type StoreInfo struct {
+	LastUpdated     time.Time
+	DownloadSha512  string
+	DownloadSize    int64
+	AnonDownloadURL string
+	DownloadURL     string
```

Comment:
they are both always set atm afaik, we use one or the other mostly based whether we have a token or not in download itself